UnitsNet/Serialization/Contracts/QuantityValueContract.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Runtime.Serialization;
+
+namespace UnitsNet.Serialization.Contracts
+{
+    /// <summary>
+    ///     Can be used to represent a quantity using a different type-representation (e.g. replacing the Enum type with it's
+    ///     string representation)
+    /// </summary>
+    /// <typeparam name="TValue">The type used to represent the quantity value (e.g. double / decimal / string) </typeparam>
+    /// <typeparam name="TUnit">The type used to represent the quantity unit (e.g. enum / int / string) </typeparam>
+    [DataContract]
+    public class QuantityValueContract<TValue, TUnit>
+    {
+        /// <summary>
+        ///     Initializes a new quantity with the given unit and value
+        /// </summary>
+        /// <param name="value">The value</param>
+        /// <param name="unit">The associated unit</param>
+        public QuantityValueContract(TValue value, TUnit unit)
```

Comment:
Isn't a default empty constructor required for WCF contracts?

Reply:
To be honest, I had never seen a constructor on a DataContract before - but the DC serializers don't seem to mind it.
I haven't tried generating any proxies - so there is still a chance this could fail. (I know you definitely need an empty constructor on the service)

Reply:
As long as it works. The tests seems to work at least.